### PR TITLE
Add `bin/banner` script for random banner generation

### PR DESCRIPTION
**Pull request by kenodegard**
### Description

Instead of hunting around for free/permissive icons and generic/stock images to use for our blog posts lets just generate something random.

This is (shamelessly) inspired by GitHub's blog (https://github.blog/) where most of the banners are some abstract gradient with the GitHub logo.

This script relies on the https://github.com/qrohlf/trianglify library and is intended to be be invoked as:

```bash
$ npm run -- banner --output static/img/blog/.../banner.png
```

> **Warning**
> One of the dependencies (`canvas`) is not compiled for macOS-arm64, you'll need to follow these instructions: https://www.npmjs.com/package/canvas#compiling

This will generate images like the following:

<details>
<summary>Version 1</summary>

</details>

### CODE REVIEW

File: bin/backdrop/backdrop.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+const output = join("..", "..", options.output);
+
+// Randomize color function
+const colorFunctions = Object.values(trianglify.colorFunctions);
+const colorFunction = colorFunctions[Math.floor(Math.random() * colorFunctions.length)];
+
+// Generate a canvas using trianglify
+const canvas = trianglify({
+  width: 1200,
+  height: 630,
```
Comment (kenodegard) [inline bin/backdrop/backdrop.js:28]: What is the "correct" size for a banner?

Comment (travishathaway) [reply bin/backdrop/backdrop.js:28]: The exact size doesn't matter, but it should be close to a 2:1 aspect ratio

File: src/components/NewsCard/index.jsx [hunk 2]
```diff
@@ -3,24 +3,17 @@ import styles from './styles.module.css';
...
-    image = (
-    <div>
-      <Image img={props.imageUrl} alt={`Banner image for ${props.title}`} className={styles.news_card_image} /> 
-      <hr style={{backgroundColor: "var(--ifm-color-secondary-lightest"}} />
-    </div>)
-  } else if (props.index === 0) {
-    image = (
-      <CondaCSVG />
-    )
-  }
```
Comment (kenodegard) [inline src/components/NewsCard/index.jsx:23]: This is why we get the big C if no image is defined in the frontMatter.

Comment (kenodegard) [reply src/components/NewsCard/index.jsx:23]: Opted to remove this, it looks cleaner IMO to display the image if present and to have no image otherwise

**Review by beeankha — APPROVED** [review]
This is great, thank you so much @kenodegard !

**Review by travishathaway — CHANGES_REQUESTED** [review]
@kenodegard,

The layout of the cards on the front page doesn't look quite right yet. The addition of the images for the other cards causes the big green card to be a lot taller and creates a lot of empty space.

Another problem with the layout of the smaller cards is the unequal height of the images. This makes everything look out of alignment.

**Review by travishathaway — APPROVED** [review]

**Pull request merged**